package.json

```diff
@@ -34,6 +34,13 @@
     "hapi": "^8.6.1",
     "bluebird": "^2.9.30",
     "apparition": "^1.0.0",
-    "chai": "^3.0.0"
+    "chai": "^3.0.0",
+    "genesis": "^1.0.0",
+    "lodash": "^2.4.2",
+    "joi": "^6.5.0",
+    "mongodb": "^2.0.35",
+    "uuid": "^2.0.1",
+    "sinon": "^1.15.4",
+    "sinon-as-promised": "^4.0.0"
```

Comment:
Many of these packages are only used for testing and should be listed as `devDependencies` rather than as `dependencies`. See https://docs.npmjs.com/files/package.json#devdependencies for more info.